### DISCUSSION TITLE
Como empezar desde cero

### DISCUSSION THREAD

**Discussion by D-a-v-i-d-Vargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#157] [Q&A] [answered]
Hola a todos agradeceria si me pudieran ayudar, tuve un percance automivilistico con mi familia y no me pude conectar la ultima semana, apenas estoy viendo todos los videos, pero de verdad no tengo ninguna idea de como realizar las katas, y me quede mucho con el modelo de nuestra primera etapa que había mas instrucciones, solo quiero ponerme al corriente para entregar todo antes del día 4.

p.d. he observado que esta parte se centra mas en investigar al cliente y como usaria nuestra pagina o la app, pero nuevamente no entiendo como hacer las katas o si necesitamos diseñar algo y apenas empece desde cero así que tengo miedo y estoy bastante confundido.

Finalmente espero puedan ayudarme y gracias por leer.

**Comment by theblower0** [comment, marked as answer]
Hola @D-a-v-i-d-Vargas  primero que todo, espero que tú y tu familia estén bien. Sobre la katas, es importante recalcar que no hay una forma "predeterminada" de hacerlas y tenemos la oportunidad de ir agregando lo que nosotros pensemos que es importante añadir. Explicaré una por una, espero se entienda bien.

1.- Requerimientos
En esa parte el mission commander compartió un documento de ejemplo, si gustas, puedes llenarlo todo o puedes solo tomar las partes que creas que son más importante, o también, hacer una nueva por tu cuenta. Aquí lo importante es que simules una forma en como tú, podrías realizar una toma de requerimientos.

2.- Buyer persona
Para crear el buyer persona, debo decir que para mi fue algo complicado, pero investigando un poco vas entendiendo como funciona. Para mi, básicamente es una persona dentro de tu publico objetivo, digamos, describiendo un "target" más a profundidad. Por ejemplo, si el publico objetivo es de mujeres entre 20-25 años, que perciben un sueldo de más de 6 mil pesos al mes, bueno, el buyer persona es Estefanía Pérez de 22 años de edad, que trabaja como cajera en un super, sus objetivos son poder subir de puesto, comprarse una moto y terminar sus estudios profesionales.

En esta página puedes crearlo
https://www.hubspot.es/make-my-persona

3.- Publico objetivo
Como te comentaba, el publico objetivo es ya una "porción" más amplia de los clientes a los que va dirigido al modelo de negocio, en esta caso abogabot. En la plantilla de miro te facilita mucho las cosas, en la demografía puedes incluir sexo, edad, estado civil, etc. En psicografia puedes colocar intereses, valores, actividades que realizan, etc.

4.- wireframe
De igual manera, en miro puedes checar las plantillas, yo usé esta 
https://miro.com/app/dashboard/?tpTemplate=website-wireframing&isCustom=false&invite_link_id=169582150993

Básicamente es el diseño UX, más o menos de como quieres la **estructura** de tu página, aquí solo representas que cosa iría en cada lugar.

5.-Diseño UI
Aquí ya le metes con toda tu creatividad que tipo de imágenes, logo, colores, etc. Colocas todo lo bonito en la estructura que diseñaste con el wireframe.

Puedes ir checando en las contribuciones de otros compañeros que compartieron sus repos para darte una idea más clara. Lo que vas a subir a tu repo, no importa el formato, si es un doc, pdf, imagen, con que tenga lo que te están pidiendo, no hay problema, espero haberte ayudado, saludos.

**Reply by OscarIslasH** [reply]
Como informacion adicional: Puedes hacer la parte del publico onjetivo tambien con Miro, el link de la plantilla es esta: https://miro.com/es/plantillas/publico-objetivo/

 Abajo explica cada una de las cosas, es como un tutorial de como irlas rellenando, solo que va enfocado todo al cliente, es decir el que va a hacer las demandas, el publico objetivo tanto el buyer persona es acerca del cliente de los abogados que al final de cuentas van a ser quienes usen el sitio web y la aplicacion para moviles. 

Si tienes dudas de como hacer un Wireframe, puedes ver como referencia el que estoy haciendo, ya casi acabo, lo que debe de llevar es lo que viene en el readme de el repositorio

El wareframe que estoy haciendo: https://miro.com/app/board/uXjVOJwm_Ic=/?invite_link_id=246275871172

El repositorio: https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO/practicas


Cualquier duda que tengas puedes preguntar sin problemas, tambien estuve sin tiempo por trabajo y ya comprendi como hay que ir haciendo las cosas.


El wireframe debe ser de un sitio web y una app movil, lo explica en el punto del readme donde dice "La página debe de ser responsive para poderla ver desde el celular." y "responsive" significa que es "accesible y adaptable para otros dispositivos"

**Comment by khisus19** [comment]
Compañero puedes visitar mi repo para que tengas una idea de como entregar tus katas. Yo lo hice sencillo pero al menos entregue un poco de cada cosa. Recuerda que son cinco puntos. Los requerimientos, el buyer persona, el público objetivo, el ux (wireframes) y el user interface. El MC dió mucha libertad en lo que vas a entregar
`https://github.com/khisus19/LaunchX-PrimeraSemana`
